### DISCUSSION TITLE
I need to search a paragraph of text to identify it.

### DISCUSSION THREAD

**Discussion by bbzcx** [jertel/elastalert2#973] [Q&A] [answered]
message:"Found no process plan tracking data for substrate" 
When I use the above method, I can search in elk

However, I will report an error when configuring rules like this in elastalert!!
filter:
- query:
    query_string:
      query: "message: "Found no process plan tracking data for substrate""

**Comment by nsano-rururu** [comment, marked as answer]
@bbzcx 

Read the guidelines!
https://github.com/jertel/elastalert2/discussions/11

**Reply by nsano-rururu** [reply]
https://elastalert2.readthedocs.io/en/latest/recipes/faq.html#how-to-write-a-query-filter-for-phrases-containing-spaces
